### PR TITLE
Eventer main function

### PR DESCRIPTION
**Pull request by mwielgus**
cc: @piosz

### CODE REVIEW

File: Makefile [hunk 1]
```diff
@@ -13,6 +13,7 @@ deps:
 build: clean deps
 	GOOS=linux GOARCH=amd64 CGO_ENABLED=0 godep go build ./...
 	GOOS=linux GOARCH=amd64 CGO_ENABLED=0 godep go build -o heapster k8s.io/heapster/metrics
+	GOOS=linux GOARCH=amd64 CGO_ENABLED=0 godep go build -o eventer k8s.io/heapster/eventes
```
Comment (gmarek) [inline Makefile]: s/eventes/events/

File: events/eventer.go [hunk 2]
```diff
@@ -0,0 +1,113 @@
...
+		glog.Fatal(err)
+	}
+	if len(sources) != 1 {
+		glog.Fatalf("Wrong number of sources: %d", len(sources))
+	}
+
+	// sinks
+	sinksFactory := sinks.NewSinkFactory()
+	sinkList := sinksFactory.BuildAll(argSinks)
+	for _, sink := range sinkList {
```
Comment (gmarek) [inline events/eventer.go]: Generally it's more efficient to use `for i := range bla { bla[i] }` as this way you're avoiding making copies. I guess that Sinks are small enough for it to doesn't matter though.

File: events/eventer.go [hunk 3]
```diff
@@ -0,0 +1,113 @@
...
+	setMaxProcs()
+	glog.Infof(strings.Join(os.Args, " "))
+	glog.Infof("Eventer version %v", version.HeapsterVersion)
+	if err := validateFlags(); err != nil {
+		glog.Fatal(err)
+	}
+
+	// sources
+	if len(argSources) != 1 {
+		glog.Fatal("wrong number of sources specified")
```
Comment (gmarek) [inline events/eventer.go]: Add "Requires exactly one source"

File: events/eventer.go [hunk 4]
```diff
@@ -0,0 +1,113 @@
...
+	if err != nil {
+		glog.Fatal(err)
+	}
+	manager.Start()
+
+	glog.Infof("Starting eventer")
+	<-quitChannel
+}
+
+func validateFlags() error {
```
Comment (gmarek) [inline events/eventer.go]: Maybe 'validateFlagValues'? My first thought was that it's some flag-library thingy.

**Comment by gmarek** [comment]
One bigger issue and few nits. Otherwise LGTM.

File: events/eventer.go (hunk 2 above)
Comment (mwielgus) [inline events/eventer.go]: They are interfaces/pointers.

File: Makefile (hunk 1 above)
Comment (mwielgus) [inline Makefile]: Done. Need to change this file so i have the same on mac and linux.

File: events/eventer.go (hunk 3 above)
Comment (mwielgus) [inline events/eventer.go]: Done

File: events/eventer.go (hunk 4 above)
Comment (mwielgus) [inline events/eventer.go]: Ack.

**Comment by gmarek** [comment]
I guess that heapster-integration errors are still expected. LGTM in such case.

**Pull request merged**